source/agora/utils/Log.d

```diff
@@ -115,3 +115,57 @@ public class AgoraLayout : Appender.Layout
         }
     }
 }
+
+// Log Level Definition for C++ code
+enum LogTrace = Level.Trace;
+enum LogInfo = Level.Info;
+enum LogWarn = Level.Warn;
+enum LogError = Level.Error;
+enum LogFatal = Level.Fatal;
```

Comment:
That doesn't seem to be used / useful ? Those definition are only visible from D.